pymc_marketing/mmm/base.py

```diff
@@ -374,11 +377,17 @@ def compute_channel_contribution_original_scale(self) -> DataArray:
             coords=channel_contribution.coords,
         )
 
-    def plot_contribution_curves(self) -> plt.Figure:
+    def plot_contribution_curves(
+        self, estimators: Optional[bool] = True
```

Comment:
Can you please add some doctrings to this method? I think users might find it useful (specially the meaning of the `estimators` argument)